dev-python/randomize/randomize-0.14.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{4,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Randomize the order of the tests within a unittest.TestCase class"
+HOMEPAGE="https://github.com/nloadholtes/nose-randomize"
+SRC_URI="https://github.com/nloadholtes/nose-randomize/archive/master.zip"
+S="${WORKDIR}/nose-${PN}-master"
+
+
+LICENSE="GNU LGPL"
```

Comment:
Which version of the LGPL? Also it should be a file in the [licenses directory](https://github.com/gentoo/gentoo/tree/master/licenses), e.g. `LGPL-3+`